static/docs/user-guide/update-DVC-pipelines.md

```diff
@@ -1,21 +1,46 @@
-# How to update your pipeline by editing dvc files
+# Updating pipelines
 
-> 1)using `dvc move` to edit file paths of outputs or dependencies.
-> 2)editting by hand the outputs paths.
-> 3)`dvc run` to make edits in the dependency files and `dvc commit` to save
-> changes.
+If you need to rename or relocate the file paths and/or dependencies of a
+specific DVC-file, then you can 
 
-If your source file is a data file, then use dvc move:
+- use `dvc move` to edit file paths of outputs or dependencies.
+- `dvc run` to make edits in the files and/or dependencies and `dvc commit` to
```

Comment:
dvc run by default does not require `dvc commit`

Reply:
The changes made by ``dvc run`` are stored in cache and after using ``dvc commit`` the stage is saved. I added it to save the present stage of files and/or dependencies and outputs.